src/config/config.go

```diff
@@ -13,6 +13,7 @@ import (
 
 var (
 	Version           = "undefined"
+	Name              = "undefined"
```

Comment:
Why is this needed here?
All the other values here are initialized with the `-X` directive here:
https://github.com/F1bonacc1/process-compose/blob/4397f98dcf8999696352e6c5d6f2292e71967707/Makefile#L12-L18

Reply:
Removed